src/components/CTypeDetails/CTypeDetails.astro

```diff
@@ -99,7 +99,7 @@ const { w3nOrigin, indexer } = configuration;
     <legend>Registration Block</legend>
 
     <a
-      href={`https://polkadot.js.org/apps/?rpc=wss%3A%2F%2F${indexer.polkadotRPCEndpoint}#/explorer/query/${block}`}
+      href={`https://polkadot.js.org/apps/?rpc=${blockchainEndpoint}#/explorer/query/${block}`}
```

Comment:
I highly recommend against building URLs this way. I was not pushing against it when you were only adding ASCII characters previously. Now when you inject special characters, you really should do either `encodeURIComponent` or build the URL using the `URL()` API.